.github/workflows/deploy_to_environment_azure.yml

```diff
@@ -121,51 +132,12 @@ jobs:
               --image ${{ env.ACR_URL }}/${{ env.DOCKER_IMAGE }}:${{ env.DOCKER_IMAGE_TAG }} \
               --output none
 
-  # This has been disabled so that existing Github workflow tags are not overwritten
```

Comment:
Can I just check - is this commented out because we aren't quite ready to start using this yet?